src/Sylius/Bundle/AddressingBundle/Controller/ProvinceController.php

```diff
@@ -86,7 +93,9 @@ protected function createProvinceChoiceForm(CountryInterface $country): FormInte
 
     protected function createProvinceTextForm(): FormInterface
     {
-        return $this->get('form.factory')->createNamed('sylius_address_province', TextType::class, null, [
+        $formFactory = $this->get('form.factory');
```

Comment:
I'm researching why this is happening, will update you later this week.

According to `phpstan-symfony` it should have been handled automatically
> Provides correct return type for ContainerInterface::get() and ::has() methods.